tests/test_lab.py

```diff
@@ -16,6 +17,26 @@ def test_no_lab(self):
         pass
 
 
+class TestNoClass(BBTestCase):
+
```

Comment:
This test code is doing something special and I'm not sure I follow. Please add a comment explaining what it tests.

Reply:
@shmir I'm merging the PR, but please add a comment